Kubernetes/container-azm-ms-agentconfig.yaml

```diff
@@ -90,6 +90,16 @@ data:
         #fieldpass = ["metric_to_pass1", "metric_to_pass12"]
 
         #fielddrop = ["metric_to_drop"]
+  mdm-metrics-configuration-settings: |-
+    # MDM metrics configuration settings for resource utilization
+    [mdm_metrics_configuration_settings.resource_utilization]
```

Comment:
make name more intuitive and explicit - common format so that it can be reused across future custom config
metricname_target(resource)